exercises/list-ops/canonical-data.json

```diff
@@ -0,0 +1,147 @@
+{
+  "#": [
+    "Though there are no specifications here for dealing with large lists,",
+    "implementers may add tests for handling large lists to ensure that the",
+    "solutions have thought about performance concerns."
+  ],
+  "append": {
+    "description": "append entries to a list and return the new list",
+    "cases": [
+      {
+        "description": "empty lists",
+        "list1": [],
+        "list2": [],
+        "expected": []
+      },
+      {
+        "description": "empty list to list",
+        "list1": [],
+        "list2": [1,2,3,4],
+        "expected": [1,2,3,4]
+      },
+      {
+        "description": "non-empty lists",
+        "list1": [1,2],
+        "list2": [2,3,4,5],
+        "expected": [1,2,2,3,4,5]
+      }
+    ]
+  },
+  "concat": {
+    "description": "concat lists and lists of list into a new list",
+    "cases": [
+      {
+        "description": "empty list",
+        "lists": [],
+        "expected": []
+      },
+      {
+        "description": "list of lists",
+        "lists": [[1,2],[3],[],[4,[5,6]]],
```

Comment:
That's funny.  I also didn't expect it to be a flatten operation either but the "examples" that i looked at in a couple languages included that.  

So the actual test here should be something more like
`[ [1,2,3], [], [4, 5] ] = [[1,2,3,4,5]]`
right?

This is "concat all the lists you handed me in this list of lists but don't flatten them."